glustercli/cmd/snapshot-info.go

```diff
@@ -47,7 +47,7 @@ func snapshotInfoDisplay(snap api.SnapGetResp) {
 	fmt.Println("Snapshot Volume ID:", vol.ID)
 	fmt.Println("State:", vol.State)
 	fmt.Println("Origin Volume name:", snap.ParentVolName)
-	fmt.Println("Snap Creation Time:", snap.SnapTime)
+	fmt.Println("Snap Creation Time:", snap.CreatedAt.String())
```

Comment:
Is there any particular reason to remove the time formatting?

Reply:
@rafikc30 added time formating